optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -306,8 +306,16 @@ def __call__(self, trial: optuna.trial.Trial) -> float:
         elapsed_secs = time.time() - start_time
         average_iteration_time = elapsed_secs / len(val_scores)
 
+        if self.model_dir is not None and self.lgbm_params.get("return_cvboost"):
```

Comment:
ditto.
```suggestion
        if self.model_dir is not None and self.lgbm_params.get("return_cvbooster"):
```

Reply:
In addition, I found that `self.model_dir` is always `None`. Could you add `model_dir` argument to `OptunaObjectiveCV.__init__` like `OptunaObjective.__init__`?